### ISSUE TITLE
Update datafoodconsortium website with a new sub menu on "achievments"

### ISSUE THREAD

**Issue by myriamboure** [datafoodconsortium/datafoodconsortium.github.io#3]
Link in it the owl files (the François is uploading on the dfc domain in parallel) + the gitbook documentation + the live prototype and github project board.

**Comment by myriamboure** [comment]
Ok I'm bad at coding, I'm fighiting to add the new menu "achievements" in the header, but I managed to add the sections in English and French page. @pacodelaluna as soon as you have the links for the files hosted in the datafoodconsortium domain I can add those links. Also @simonLouvet when you have the link for a live accessible version of the prototype people can start playing with I can add it. 

I asked @arthur-koenig who did the website how to add the new section in the menu, waiting for his answer.

**Comment by simonLouvet** [comment]
http://185.25.192.252:8080/

**Comment by myriamboure** [comment]
Ok finally I have added the prototype live link in the Gitbook in the "status of the project" page but not directly on the website as still hard to manipulate I think... Also the owl links, from Gitbook we send people directly to Github which I think is fine for now, so I won't do any link from the website directly. Just missing the menu, I asked some help from @pacodelaluna

**Comment by pacodelaluna** [comment]
@myriamboure Just fixed it, it is live.

**Comment by myriamboure** [comment]
Awesome, thanks @pacodelaluna ! it's ready for review and closing then :-)
